uberenv.py

```diff
@@ -150,6 +150,11 @@ def parse_args():
                       dest="package_name",
                       default=None,
                       help="override the default package name")
+    # overrides uberenv_package_name
+    parser.add_option("--uberenv-package-name",
```

Comment:
How is this different from --package-name that already exists?

Reply:
its presence triggers non dev build for the "non install" case (like ancient ubrerenv)

Reply:
--force-spack-install? 

then you would end up with `spack --package-name=fakepackage --force-spack-install` or just `spack --force-spack-install` which would end up with default package for the project.

My concern is how is a user supposed to know the difference between:

spack --package-name=foo
spack --uberenv-package-name=foo

without digging into the code.

Reply:
Better names are certainly welcome.

I think this will be set only in the json file for the post part. 

use case is to have uberenv work out of the box to build tpls when the `--install` option is not passed.

Having the option only be in the json file might be another option?

Reply:
Engage-Cyrus-Mode? =)

Reply:
I am fine with removing this as an option from the command line.